dialoptions.go

```diff
@@ -210,20 +210,30 @@ func WithReadBufferSize(s int) DialOption {
 // WithInitialWindowSize returns a DialOption which sets the value for initial
 // window size on a stream. The lower bound for window size is 64K and any value
 // smaller than that will be ignored.
+//
+// Deprecated: use WithInitialStreamWindowSize to set a stream window size without disabling
+// dynamic flow control.
+// Will be supported throughout 1.x.
```

Comment:
nit: This will not be rendered in a new line. I think we can merge it with the previous line.